### DISCUSSION TITLE
Error on serverpod generate in generated file in server folder

### DISCUSSION THREAD

**Discussion by sandeepdevacc** [serverpod/serverpod#522] [Q&A] [answered]
Missing concrete implementations of 'SerializableEntity.toJson' and 'TableRow.toJsonForDatabase'.
Try implementing the missing methods, or make the class abstract.

@yahu1031  @II11II @vlidholt

**Comment by yahu1031** [comment]
@sandeepdevacc thanks for issues bruh. But just don't post the error msg only. Please provide the sufficient details of the error. Like code blocks, what you have done and tried, etc...

**Comment by sandeepdevacc** [comment]
> 

@yahu1031  thanks for your reply 

![image](https://user-images.githubusercontent.com/118791435/209616569-db0232a3-1410-496d-915a-95d2301f7811.png)

this is the issue got when I run serverpod generate command in my server project.

**Comment by yahu1031** [comment]
Try deleting the generated folder and generating it again.

**Comment by sandeepdevacc** [comment]
It's not working @yahu1031  
Is there any other way?

**Comment by vlidholt** [comment, marked as answer]
It's possible that you have a missmatch between the version of the `serverpod` cli tool and the serverpod packages referenced in your project. Run `serverpod version` and check that it matches the version in your `pubspec.yaml` files. There may have been some breaking changes since you created your project, see the [changelog](https://pub.dev/packages/serverpod/changelog) for instructions on how to update your project.

**Comment by sandeepdevacc** [comment]
![Screenshot 2023-01-04 at 4 29 04 PM](https://user-images.githubusercontent.com/118791435/210540980-309c9016-e37d-444b-8ed2-4f7118230847.png)

This is the screenshot of my serverpod version and pubspec.yaml file @vlidholt 

Flutter version ---> 3.3.10

**Reply by yahu1031** [reply]
@sandeepdevacc please upgrade the serverpod version to `0.9.21`. and upgrade all serverpod dependencies to the same version.
